.cicdtemplate/.github/workflows/review_pull_request.yml

```diff
@@ -38,22 +39,22 @@ jobs:
         run: ./gradlew detekt
 
       # Run Android Lint and generate Lint reports
-      - name: Run Android Lint on RxJavaTemplate
-        working-directory: ./RxJavaTemplate      
+      - name: Run Android Lint on CoroutineTemplate
+        working-directory: ./CoroutineTemplate
         run: ./gradlew lint
 
-      - name: Run Android Lint on CoroutineTemplate
-        working-directory: ./CoroutineTemplate      
+      - name: Run Android Lint on RxJavaTemplate
+        working-directory: ./RxJavaTemplate
         run: ./gradlew lint
 
-      # TODO: Update to support test coverage report from Jacoco for Coroutine Template
-      # Run Unit Test & Jacoco for code coverage for RxJavaTemplate
-      - name: Run unit tests & Jacoco on RxJavaTemplate
+      # TODO: Update to support test coverage reports from Jacoco for CoroutineTemplate
+      # Run unit tests and Jacoco for code coverage for RxJavaTemplate
+      - name: Run unit tests and Jacoco on RxJavaTemplate
```

Comment:
🤔  This is the ci cd template so I think we could get rid of `RxJavaTemplate` or `CoroutineTemplate`, we should use general words as much as possible IMO 😄  And we don't need to keep both steps `Run Android Lint` because we only use one step to execute lint check on client projects

Reply:
I agree with @hoangnguyen92dn . Sadly, I recalled I tried before and failed 🙈

It would be great if we can find a way to group them. But if it's too difficult, I would suggest keeping it for a different pull request as it's a bit out of scope 🚀

Reply:
@Tuubz This file is located in `cicdtemplate` folder so it doesn't affect the CI 😄 , we still need to keep the current workflow applied for the `android-templates` 😄

Reply:
Updated 🙏🏼